### ISSUE TITLE
[Explore] Colors for log level not working when level is a label instead of a parsed field

### ISSUE THREAD

**Issue by olivierboudet** [grafana/grafana#21112]
**What happened**:
I have some Java logs stored in loki with fluent bit plugin. Fluent bit parses the logs to extract a level field.
Then in grafana the logs appears with different colors corresponding to the level (debug in blue for example). 
![image](https://user-images.githubusercontent.com/857205/70895044-7abb6380-1fee-11ea-81e8-32c28f417110.png)

When I change the fluent bit configuration to add the level as a label in loki to be filterable, the colors disappears, all the logs are displayed with level "unknown" as on the following screenshot : 

![image](https://user-images.githubusercontent.com/857205/70894987-5e1f2b80-1fee-11ea-81e7-a9cd4a2eddc9.png)

The fluent bit configuration for loki which is not working is as below : 
```
labelmap.json: |-
    {
      "kubernetes": {
        "container_name": "container",
        "host": "node",
        "labels": {
          "app": "app",
          "type": "type",
          "release": "release"
        },
        "namespace_name": "namespace",
        "pod_name": "instance"
      },
      "stream": "stream",
      "level":"level"
    }
```

**What you expected to happen**:

I expect that colors are correctly displayed whenever the level is a parsed field or a label.
Moreover, I do not see anything in the documentation about this feature.

**Environment**:
- Grafana version: 6.0.5
- Data source type & version: Loki 1.2.0 & fluent bit plugin 0.1 (docker image grafana/fluent-bit-plugin-loki:0.1)
- OS Grafana is installed on: GKE
- User OS & Browser: Linux & Chromium 78
- Grafana plugins: /
- Others:

**Comment by roobert** [comment]
Came here to open this same issue. I've observed the following:

* Parsing the `level` out of the log message appears to work as expected:
<img width="904" alt="Screen Shot 2020-01-12 at 12 29 48" src="https://user-images.githubusercontent.com/226654/72218852-9a837000-3537-11ea-8652-bdba754078db.png">

* Specifying a `label` results in the log message being type "unknown":
<img width="903" alt="Screen Shot 2020-01-12 at 12 28 31" src="https://user-images.githubusercontent.com/226654/72218851-9a837000-3537-11ea-8746-baa8f46d95aa.png">

* Specifying a `label` and having `level` in the log message, whether they conflict or are the same, results in the message being type "unknown":
<img width="907" alt="Screen Shot 2020-01-12 at 12 31 22" src="https://user-images.githubusercontent.com/226654/72218853-9a837000-3537-11ea-949f-5215f2514e1e.png">

Hope this helps!

**Comment by ivanahuckova** [comment]
@roobert @olivierboudet thanks so much for creating this issue! 🙏 Fix will be part of 6.6.0-stable release.

**Comment by remithomasn7** [comment]
I am currently using Grafana v7.0.6, and I am still facing the issue. To be more precise, when I have no label for my log level, or a label which is not 'level' but 'lvl'. Grafana is (almost) able to set a level and a color to my log:

![Screenshot 2020-07-21 at 15 22 59](https://user-images.githubusercontent.com/45794704/88060173-253bcc00-cb66-11ea-834c-e9881611e62f.png)

I said almost, cause as you can see on the screenshot above, for one of my log line, the lvl=INF is set to unknown. That's a first issue but not the major one.

When I use a label named 'level', Grafana is facing more trouble to identify my log level (and to set a proper color):
![Screenshot 2020-07-21 at 15 27 33](https://user-images.githubusercontent.com/45794704/88060602-c0cd3c80-cb66-11ea-8dce-72f02d03d51d.png)

This time, only the log line with 'lvl=ERR' was well identified. As this issue is closed I am wondering if I am missing something, or if this issue is still there.
